src/shared/GlobalTopBanners/OktaBanners/OktaBanners.tsx

```diff
@@ -0,0 +1,31 @@
+import { lazy } from 'react'
+import { useParams } from 'react-router'
+
+import { useOktaConfig } from 'pages/AccountSettings/tabs/OktaAccess/hooks'
+
+interface URLParams {
+  provider: string
+  owner?: string
+}
+
+const OktaEnabledBanner = lazy(() => import('../OktaEnabledBanner'))
+const OktaEnforcedBanner = lazy(() => import('../OktaEnforcedBanner'))
+
+function OktaBanners() {
+  const { provider, owner } = useParams<URLParams>()
+
+  const { data } = useOktaConfig({
+    provider,
+    username: owner || '',
+    opts: { enabled: !!owner },
+  })
+
+  const oktaConfig = data?.owner?.account?.oktaConfig
+
+  if (!owner || !oktaConfig?.enabled || !data?.owner?.isUserOktaAuthenticated)
```

Comment:
nit: can potentially move the !owner call to before the hook if we want to avoid calling the hook

Its a slight perf upgrade, not sure how much its worth in practice

Reply:
Can't unconditionally call a hook!